transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/EncodeTransformationPipelineBuilder.scala

```diff
@@ -85,6 +88,22 @@ object EncodeTransformationPipelineBuilder extends PipelineBuilder[Args] {
     ()
   }
 
+  /** Message value used by ENCODE in place of null. */
+  val UnknownValue: Msg = Str("unknown")
```

Comment:
Should the "U" at the start of `UnknownValue` be lower-case for variable naming convention?

Reply:
This aligns with the convention of using UpperCamelCase for constants. It's also required to be this for practical reasons, because if it's lower-cased then you can't match on it as an equality check on the LHS of a `match` statement (if it's lower-cased, the parser views it as a new name generated by the pattern-match, not a reference to a constant).